web/impact/impact/v1/views/criterion_list_view.py

```diff
@@ -11,3 +11,13 @@ class CriterionListView(BaseListView,
     view_name = "criterion"
     helper_class = CriterionHelper
     actions = ['GET', 'POST']  # Should get this from PostMixin
+
+    def get(self, request):
+        self.validate_id(request, 'judging_round_id')
+        return super().get(request)
+
+    def filter(self, qs):
+        return self._filter_by_judging_round_id(qs)
```

Comment:
This should be calling super() to pick up the other standard filters.  Something like:
`return self._filter_by_judging_round_id(super().filter(qs))`